test/scripts/xpost.js

```diff
@@ -23,70 +23,89 @@ const robot = {
 
 describe('xpost', () => {
   beforeEach(() => {
-    mockSandbox.reset();
+    mockSandbox.resetBehavior();
+    mockSandbox.resetHistory();
```

Comment:
A sinon API change.  These are the only two lines I manually changed.  I guess my code formatter got the rest of it.  😛